io_scene_godot/converters/material/material.py

```diff
@@ -57,8 +57,57 @@ def export_material(escn_file, export_settings, bl_object, material):
 
 def export_as_spatial_material(material_rsc_name, material):
     """Export a Blender Material as Godot Spatial Material"""
+    out = material.node_tree.get_output_node("ALL")
```

Comment:
Maybe add a  check for `node_tree`, it can be `None` sometimes.
Other changes looks very good. Thank you very much!.